pages/alpha/sponsorship/index.tsx

```diff
@@ -0,0 +1,214 @@
+import type { NextPage } from "next";
+
+import Layout from "../../../components/Layout/Layout";
+import Image from "next/image";
+import Link from "next/link";
+import { useEffect, useState } from "react";
+
+const Sponsorship: NextPage = () => {
+  const [scroll, setScroll] = useState(0);
+
+  useEffect(() => {
+    function handleScroll() {
+      setScroll(window.scrollY);
+    }
+    window.addEventListener("scroll", handleScroll);
+
+    return () => {
+      window.removeEventListener("scroll", handleScroll);
+    };
+  }, []);
+
+  return (
+    <Layout>
+      <div>
+        <header className="bg-white text-black text-center pt-10 pb-24 px-4 sm:pb-36 sm:text-left sm:px-10 md:pb-44 md:px-20 lg:px-36 lg:pt-14">
+          <h1 className="font-extrabold text-3xl md:text-4xl md:mb-2 lg:text-5xl">
+            Become a{" "}
+            <b className="bg-clip-text text-transparent bg-gradient-to-r from-orange-400 to-pink-600 z-20 font-extrabold">
+              Sponsor
+            </b>
+          </h1>
+          <h3 className="text-lg font-bold">
+            Reach thousands of developers every mounth!
+          </h3>
+        </header>
+        <section className="flex items-center relative bottom-12 justify-center overflow-hidden gap-8 sm:gap-20 sm:bottom-20 md:bottom-24  md:gap-36 lg:gap-44">
+          <div
```

Comment:
If you could make a array and map over the images here that will make the code a little concise.
Also adding some alt text would be good.